installer/bundle/bundle_skel.sh

```diff
@@ -817,6 +834,21 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+            for i in bundles/*-bundle-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+
+                # It's possible we have a test file without bundle; if so, ignore it
+                BUNDLE=`basename $i -bundle-test.sh`
```

Comment:
ok, I thought test file here means some other input file that is used by the script which is going to look for it :) Nevermind.